packages/shopware-6-client/src/services/contextService.ts

```diff
@@ -77,7 +77,7 @@ export async function getAvailableCurrencies(): Promise<
 > {
   const resp = await apiService.get(getContextCurrencyEndpoint());
```

Comment:
```suggestion
  const {data}= await apiService.get(getContextCurrencyEndpoint());
```
Maybe destructuryzation?